src/test/java/com/example/glovo/GlovoApplicationTests.java

```diff
@@ -6,8 +6,8 @@
 @SpringBootTest
 class GlovoApplicationTests {
 
-	@Test
-	void contextLoads() {
-	}
+    @Test
+    void contextLoads() {
```

Comment:
this test do nothing

Reply:
This test was autogenerated by Idea, I added it to pull by accident. deleted